ipaclient/install/client.py

```diff
@@ -2055,9 +2055,9 @@ def install_check(options):
     global client_domain
     global cli_basedn
 
-    print("This program will set up FreeIPA client.")
-    print("Version {}".format(version.VERSION))
-    print("")
+    logger.info_cmd("This program will set up FreeIPA client.")
+    logger.info_cmd("Version {}".format(version.VERSION))
```

Comment:
pylint is going to complain about this style. You have to use old-style formatting and replace this kind of calls with ``logger.info_cmd("Version %s", version.VERSION)``.

Reply:
Fixed